include/numerics/type_n_tensor.h

```diff
@@ -46,13 +47,25 @@ template <unsigned int N, typename T>
 class TypeNTensor
 {
 public:
-  TypeNTensor () {}
-
-  TypeNTensor (const T &) {}
-
-  TypeNTensor (const TypeVector<T> &) {}
-
-  TypeNTensor (const TypeTensor<T> &) {}
+  TypeNTensor ()
+    {
+      _coords.resize(int_pow(LIBMESH_DIM, N));
+    }
+
+  TypeNTensor (const T &)
+    {
+      _coords.resize(int_pow(LIBMESH_DIM, N));
+    }
+
+  TypeNTensor (const TypeVector<T> &)
+    {
+      _coords.resize(int_pow(LIBMESH_DIM, N));
+    }
+
+  TypeNTensor (const TypeTensor<T> &)
+    {
+      _coords.resize(int_pow(LIBMESH_DIM, N));
+    }
```

Comment:
Was this a bug? How were we not resizing these before? Shouldn't this be done in the initializer lists?

Reply:
Oh I see, you just added `_coords`. So I guess this class wasn't used for anything previously?